src/main/java/gtPlusPlus/core/lib/CORE.java

```diff
@@ -321,13 +321,13 @@ public static EntityPlayerMP getFakePlayer(World world) {
 	public static class Everglades{
 		public static final String MODID = "ToxicEverglades";
 		public static final String NAME = "GT++ Toxic Everglades";
-		public static final String VERSION = "0.1";
```

Comment:
These are separate mods and shouldn’t be versioned the same as the main mod?

Reply:
I can drop these back - or since merge master already merged do you just want to update on your copy